src/planscape/forsys/views.py

```diff
@@ -278,7 +283,9 @@ def transform_into_forsys_df_data(condition_rasters: QuerySet,
 # Runs a forsys scenario sets call.
 def run_forsys_scenario_sets(
         forsys_input_dict: dict[str, list],
-        priorities: list[str]) -> ForsysScenarioSetOutput:
+        priorities: list[str], forsys_proj_id_header: str,
```

Comment:
nit: prefer one argument per row

Reply:
I'm trying to find this convention in https://google.github.io/styleguide/pyguide.html but don't see it.
We can have stricter conventions ourselves, but I also don't see this rule being followed in other files in this repo.
It's not easy to change in all files since auto-formatters don't catch it.

I'll ignore this.
If it's a strong preference, though, do you want to bring it up at a standup?

Reply:
You're right, it's not in the style guide. I still prefer it but if you don't agree I'm happy for it to stay this way